app/src/main/java/org/mozilla/fenix/components/metrics/MetricsStorage.kt

```diff
@@ -71,15 +81,15 @@ internal class DefaultMetricsStorage(
             Event.GrowthData.SetAsDefault -> {
                 settings.setAsDefaultGrowthSent = true
             }
-            Event.GrowthData.FirstAppOpenForDay -> {
-                settings.resumeGrowthLastSent = System.currentTimeMillis()
-            }
-            Event.GrowthData.FirstUriLoadForDay -> {
-                settings.uriLoadGrowthLastSent = System.currentTimeMillis()
-            }
             Event.GrowthData.FirstWeekSeriesActivity -> {
                 settings.firstWeekSeriesGrowthSent = true
             }
+            Event.GrowthData.UriLoaded -> {
```

Comment:
nit - could these be re-ordered to match the above `shouldTrack`ordering as before?